### ISSUE TITLE
Other windows 

### ISSUE THREAD

**Issue by pacav69** [Komodo/KomodoEdit#473]
When using other windows such as the help or preferences or extensions developer, when minimised or clicked out of view there is no option in menu to view this window other than right clicking on the komodo icon and selecting it. It ould be useful to have a menu selection of 'Windows' or have it under the 'View' menu.

**Comment by Naatan** [comment]
How would this be preferable over using your Window manager though? Which is optimized for exactly this behaviour. ie. I don't see clicking on a Windows menu and then selecting the relevant window being preferable over alt-tabbing or using the task bar, which both offer a uniform experience across all your apps.

I'm not against the idea, just trying to understand the merit in implementing it.

**Comment by pacav69** [comment]
I use a MAC so it was not the same setup as windows task bar, later i saw 5 or 6 windows open on the same content and i was looking for the window menu then later i tried using right click on the icon on the dock and saw so many windows open, it could be a problem for new users who are unfamiliar with the software and could potentially cause komodo to become unstable and perhaps crash. I've seen on other IDE's that they have a window menu selection that allows selection of the windows that go out of view.

**Comment by Naatan** [comment]
Fair point, I have ran into it myself a few times as well but generally my window manager comes to the rescue.
